cupyx/_ufunc_config.py

```diff
@@ -1,16 +1,51 @@
 import contextlib
 import threading
 
-config = threading.local()
-config.divide = None
-config.over = None
-config.under = None
-config.invalid = None
-# In favor of performance, the `devInfo` input/output from cuSOLVER routine
-# calls that is necessary to check the validity of the other outputs, are
-# ignored, as D2H copy incurring device synchronizations would otherwise be
-# required.
-config.linalg = 'ignore'
+_config = threading.local()
+
+
+def get_config_divide():
```

Comment:
Instead of the try/except you could rely on `getattr` with an appropriate default value:

```python
getattr(_config, 'divide', None)
```

Reply:
Yeah, I agree with this, but this approach won't set the actual value in the `_config` object unless an explicit assignment is done.
```
divide = getattr(_config, 'divide', None)
_config.divide = divide
```
I am fine with doing either both approaches. @niboshi which one do you think it favors readability?

Thanks

Reply:
@quasiben Thank you for suggestion.
I prefer try-except because it's faster than `getattr`.

Reply:
Great!  I didn't know there was a speedup with try/except